components/operations-controller/go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/kyma-incubator/compass/components/director v0.0.0-20210301143156-3b39f117e7bf
 	github.com/kyma-incubator/compass/components/system-broker v0.0.0-20210301181003-c1c76083a015
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.4
+	github.com/onsi/ginkgo v1.15.0
+	github.com/onsi/gomega v1.10.5
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.7.0
```

Comment:
Update github.com/sirupsen/logrus v1.8.0
and
golang.org/x/oauth2 v0.0.0-20210220000619-9bb904979d93